GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -1117,6 +1117,7 @@ private void UpdatePlaceholders()
                 this.blobSizes.Flush();
 
                 int deleteFolderPlaceholderAttempted = 0;
+                int deleteFolderPlaceHolder = 0;

```

Comment:
Nit: Could you rename `deleteFolderPlaceHolder` to be slightly more descriptive?  A few ideas:

- `folderPlaceholdersDeleted`
- `deleteFolderPlaceholderSucceeded`